providers/base/bin/cpuid.py

```diff
@@ -111,6 +111,7 @@
         "AMD Milan-X":      ['0xa00f12'],
         "AMD ROME":         ['0x830f10'],
         "AMD Ryzen":        ['0x810f81'],
+        "Bergamo":          ['0xaa0f01'],
```

Comment:
Since we're at this a second time, can you change the string to "AMD Bergamo" so it's consistent with the other AMD entries?

Reply:
I have updated the branch to reflect AMD Bergamo